src/JBMultiTerminal.sol

```diff
@@ -1486,12 +1486,15 @@ contract JBMultiTerminal is JBPermissioned, ERC2771Context, IJBMultiTerminal {
         // Keep a reference to the specification being iterated on.
         JBRedeemHookSpecification memory specification;
 
+        // Keep a reference to the fee for the specification's amount.
+        uint256 specificationAmountFee;
+
         for (uint256 i; i < numberOfSpecifications; i++) {
             // Set the specification being iterated on.
             specification = specifications[i];
 
             // Get the fee for the specified amount.
-            uint256 specificationAmountFee =
```

Comment:
nit: I think I like the old style here better, just because it makes it harder to make a mistake in the future and we don't need this value outside of the scope of the loop.

Reply:
got it. will change